playbooks/roles/lets-encrypt/vars/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+le_base: /var/lib/acme
+le_port: 80
+# Currently Let's Encrypt does not provide a range of IP addresses they are
```

Comment:
Let's Encrypt won't provide a range of IP addresses as a matter of policy. I think you can delete this entire comment :-)